scripts/actions/mobskills/aerial_blast.lua

```diff
@@ -9,13 +9,14 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local dmgmod = 3
```

Comment:
Is there a reason we are removing the local definition and just putting the number inside the damage function?

Reply:
Since you were going through all of them already wouldn't it of been easier to just remove it and do that revision now? I guess its kind of late for that since you're already 210 files in ha. The dmgmod I am guessing was the "ftp" and whoever started to add weapondmg * (random number) just didnt understand it as there was no real documentation from when it was originally written.

Talk for another day lol

Reply:
The purpose of this PR was to fix a very simple issue. The whole "While im here, im gonna reformat all scripts that used the original function" is just my OCD not letting me ignore the other issues.

Reviewing this PR is already gonna be a nightmare without a lot of faith, so I rahter leave that for a different PR, which will be much easier to review, because all scripts will only have a single line change